tests/.suitespec.json

```diff
@@ -16,7 +16,20 @@
             "tests/__init__.py",
             "tests/.suitespec.json",
             "tests/suitespec.py",
-            ".circleci/*"
+            ".circleci/*",
+            "tests/meta/*",
+            "tests/smoke_test.py",
+            "tests/subprocesstest.py",
+            "tests/wait-for-services.py",
+            "tests/webclient.py",
+            "tests/test_module/*",
+            "tests/lib-injection/dd-lib-python-init-test-django/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-gunicorn/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-gunicorn-alpine/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-no-perms/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-pre-installed/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-unsupported-python/*",
+            "tests/lib-injection/dd-lib-python-init-test-django-uvicorn/*"
```

Comment:
nit: Is there a reason we can't just consolidate these into `"tests/lib-injection/*"`?

Reply:
The suitespec checker script didn't like that when I tried it 🤷